src/pages/PlanPage/subRoutes/CurrentOrgPlan/BillingDetails/Address/AddressForm.tsx

```diff
@@ -22,15 +23,49 @@ function AddressForm({
   owner,
 }: AddressFormProps) {
   const elements = useElements()
+  useEffect(() => {
+    const updateAppearance = () => {
```

Comment:
do / should we change this to use the context?

Reply:
Please do

Reply:
Updated!